### PR TITLE
feat:Create prediction results migration file

### PR DESCRIPTION
**Pull request by Yuichi-Kurino**
# Administrative Info
Make sure your branch name conforms to: `feature/hotfix/.../describe_your_branch_with_underscores

### Changes
What changes did you make?
- Created a prediction_results migration file

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code

### Confirmation of Change 
The changes were made on database/migrations/20220303205124_prediction_results.js

### CODE REVIEW

**Comment by Andrew123098** [comment]
LGTM

**Review by Andrew123098 — APPROVED** [review]
Amazing code

File: database/migrations/20220303205124_prediction_results.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('prediction_results', (tbl)=>{
+        tbl.increments('id').unique().notNullable();
+        tbl.text('image_url').notNullable();
+        tbl.boolean('prediction_is_car').notNullable();
```
Comment (mingkaili) [inline database/migrations/20220303205124_prediction_results.js:6]: nit: Maybe name this is_car_prediction? bc prediction_is_car sounds kinda awkward to read? Sry, I didn't catch this while reading the design doc... might be a good idea to update the design doc too

File: database/migrations/20220303205124_prediction_results.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        tbl.boolean('true_car_prediction');
+        tbl.boolean('prediction_license_plate').notNullable();
```
Comment (mingkaili) [inline database/migrations/20220303205124_prediction_results.js:8]: Ditto, license_plate_prediction?

**Review by mingkaili — CHANGES_REQUESTED** [review]
2 small nits, besides that, make sure to create an issue and link it to this Pull request please. Just do it like how we talked about during the meeting.

After that, make sure to check the [Project board](https://github.com/orgs/Proko-Tech/projects/1) and put it in the review/blocked column. Thanks!

File: database/migrations/20220303205124_prediction_results.js (hunk 1 above)
Comment (Yuichi-Kurino) [reply database/migrations/20220303205124_prediction_results.js:6]: got it, I'll make that change in both the codebase and design doc

File: database/migrations/20220303205124_prediction_results.js (hunk 2 above)
Comment (Yuichi-Kurino) [reply database/migrations/20220303205124_prediction_results.js]: done

File: database/migrations/20220303205124_prediction_results.js (hunk 1 above)
Comment (Yuichi-Kurino) [reply database/migrations/20220303205124_prediction_results.js]: done

**Review by mingkaili — APPROVED** [review]
LGTM!

**Pull request merged**